### PR TITLE
New JSON library added

### PR DESCRIPTION
**Pull request by HSadeghein**
Please take look at [7th Issue ](https://github.com/mbuet2ner/JEasyCrypto/issues/7)
We have only tested the project on Ubuntu. 
The build and start bash files have been changed just for the Linux version.
This issue was introduced on [Merge pull request #31](https://github.com/mbuet2ner/JEasyCrypto/pull/31) from kivelas/tryCatchCorrection

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Repository includes four components:
 * **JEasyCryptoServer**, which provides a way to use the Lib from internet, using UDP and JSON.
 * **JEasyCryptoClient**, which uses the Server over network for en-/decryption of text.
 
-Additionally, the code uses the **json-simple** library (see below).
+Additionally, the code uses the **JSON-java** library (see below).
```
Comment (mbuet2ner) [inline README.md]: Could you please add the license of the JSON library?

Comment (HSadeghein) [reply README.md]: Like adding a link to the License page in the Readme file?

Comment (mbuet2ner) [reply README.md]: Yes good idea. Maybe something like 

> the JSON-java library is licensed under XXX (with clickable link)

Comment (HSadeghein) [reply README.md]: Thanks for the response. It is done.

**Pull request merged**